external/mcpclient/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@microsoft/spark.mcpclient",
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "homepage": "https://github.com/microsoft/spark.js",
+  "bugs": "https://github.com/microsoft/spark.js/issues",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/spark.js.git",
+    "directory": "packages/mcpclient"
+  },
+  "keywords": [
+    "microsoft",
+    "teams",
+    "msteams"
+  ],
+  "scripts": {
+    "clean": "npx rimraf ./dist",
+    "build": "npx tsup",
+    "test": "npx jest"
+  },
+  "peerDependencies": {
+    "@modelcontextprotocol/sdk": "1.5.0",
```

Comment:
`@microsoft/spark.common` should also be a peer dep (logging and such)

Reply:
It's okay to keep it in dev-dep right? (afaik, peer dep + dev dep is okay to be the same?) thoughts?

Reply:
peer requires them to explicitly have it installed themselves which all the other samples/packages use currently. It prevents duplicate package installations due to version differences. Its also not a dev dep since it has implementations required at runtime.